### PR TITLE
Rectified error in code snippet

### PR DESCRIPTION
**Pull request by ankit01-oss**

### CODE REVIEW

File: blog/2022-05-26-opentelemetry-kafka.md [hunk 1]
```diff
@@ -67,8 +67,8 @@ SigNoz can be installed on macOS or Linux computers in just three steps by using
 
 The install script automatically installs Docker Engine on Linux. However, on macOS, you must manually install <a href = "https://docs.docker.com/engine/install/" rel="noopener noreferrer nofollow" target="_blank">Docker Engine</a> before running the install script.
 
-```
-git clone -b main <https://github.com/SigNoz/signoz.git>
+```jsx
```
Comment (prashant-shahi) [inline blog/2022-05-26-opentelemetry-kafka.md]: @ankit01-oss  we should use `bash` or `sh` for code blocks with these commands.

File: blog/2022-05-28-distributed-tracing-golang.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ SigNoz can be installed on macOS or Linux computers in just three steps by using
 The install script automatically installs Docker Engine on Linux. However, on macOS, you must manually install <a href = "https://docs.docker.com/engine/install/" rel="noopener noreferrer nofollow" target="_blank">Docker Engine</a> before running the install script.
 
 ```jsx
```
Comment (prashant-shahi) [inline blog/2022-05-28-distributed-tracing-golang.md]: same as above

File: blog/2022-05-26-opentelemetry-kafka.md (hunk 1 above)
Comment (pranay01) [reply blog/2022-05-26-opentelemetry-kafka.md]: @prashant-shahi whats the advantage of using `sh` over `jsx` for these code snippets

Comment (prashant-shahi) [reply blog/2022-05-26-opentelemetry-kafka.md]: @pranay01 we get accurate syntax highlighting

Comment (ankit01-oss) [reply blog/2022-05-26-opentelemetry-kafka.md]: @pranay01 @prashant-shahi cool I will make the relevant change and commit.

Comment (ankit01-oss) [reply blog/2022-05-26-opentelemetry-kafka.md]: @pranay01 @prashant-shahi done.

**Pull request merged**